python/servo/package_commands.py

```diff
@@ -209,24 +209,21 @@ def package(self, release=False, dev=False, android=None, debug=False, debugger=
             msi_path = path.join(dir_to_msi, "Servo.msi")
             print("Packaged Servo into {}".format(msi_path))
         else:
+            dir_to_temp = '/'.join(binary_path.split('/')[:-2]) + '/targz'
             dir_to_package = '/'.join(binary_path.split('/')[:-1])
             dir_to_root = '/'.join(binary_path.split('/')[:-3])
-            resources_dir = dir_to_package + '/resources'
-            if os.path.exists(resources_dir):
-                delete(resources_dir)
-            shutil.copytree(dir_to_root + '/resources', resources_dir)
+            resources_dir = dir_to_temp + '/resources'
+            if os.path.exists(dir_to_temp):
+                print("Cleaning up from previous packaging")
+                delete(dir_to_temp)
             browserhtml_path = find_dep_path_newest('browserhtml', binary_path)
             if browserhtml_path is None:
                 print("Could not find browserhtml package; perhaps you haven't built Servo.")
                 return 1
```

Comment:
Please move the 'find browserhtml' section before the 'delete old `dir_to_temp`' section.